src/libraries/Microsoft.PowerFx.Connectors/Public/CdpRecordType.cs

```diff
@@ -26,19 +26,39 @@ internal CdpRecordType(ConnectorType connectorType, DType recordType, ICdpTableR
             ReferencedEntities = referencedEntities;
         }
 
-        public override bool TryGetFieldType(string name, out FormulaType type)
+        public bool TryGetFieldExternalTableName(string fieldName, out string tableName)
         {
-            if (!base.TryGetBackingDType(name, out _))
+            if (!base.TryGetBackingDType(fieldName, out _))
+            {
+                tableName = null;
```

Comment:
can move line 33 to line 30. then can remove line 41

Reply:
good idea